sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/ChatAsyncClient.java

```diff
@@ -214,6 +285,16 @@ public Builder userAgentInterceptor(Interceptor userAgentInterceptor) {
             return this;
         }
 
+        public Builder context(Context context) {
+            this.context = context;
+            return this;
+        }
+
+        public Builder userToken(String userToken) {
+            this.userToken = userToken;
+            return this;
+        }
```

Comment:
I see, so if user set `context` and `userToken` then we create `SignalingClient` which enables real time notification, I feel like, it might be bit hard for users to discover this relationship, what do you think?

How about a builder method name that better communicate this? Something like `enableRealtimeNotification(context, userToken)`